content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -47,6 +47,8 @@ Use frontend data from RUM, as well as backend, infrastructure, and log informat
 
 ### Setup RUM
 
+**Note:** Configuring RUM and Traces makes use of RUM paid data in APM, which may impact your billing.
+
```

Comment:
i think it's the opposite actually -- if you configure APM spans to connect to RUM data it impacts your billing.

my thought process is if we did this on the APM docs page the statement you added is accurate but in this case i think it's the other way around, wdyt?